components/plurals/data/macros.rs

```diff
@@ -0,0 +1,2 @@
arZeroVec :: from_bytes_unchecked (b"\x02\0\0\0\0\0\r\0\xC1\n\0\0\0\x02\0\0\0\x02\0\0\0\x81d\0\0\0\x0C\0\0\0\x0C\0\0\0") } ,)) , few : None , many : Some (icu_plurals :: rules :: runtime :: ast :: Rule (unsafe { zerovec :: VarZeroVec :: from_bytes_unchecked (b"\x02\0\0\0\0\0\x15\0\xC1\n\0\0\0\x07\0\0\0\x07\0\0\0\x08\0\0\0\x08\0\0\0\x81d\0\0\0\x11\0\0\0\x11\0\0\0\x12\0\0\0\x12\0\0\0") } ,)) , } ; static SV : < icu_plurals :: provider :: OrdinalV1Marker as icu_provider :: DataMarker > :: Yokeable = icu_plurals :: provider :: PluralRulesV1 { zero : None , one : Some (icu_plurals :: rules :: runtime :: ast :: Rule (unsafe { zerovec :: VarZeroVec :: from_bytes_unchecked (b"\x02\0\0\0\0\0\x15\0\xC0\n\0\0\0\x01\0\0\0\x01\0\0\0\x02\0\0\0\x02\0\0\0\x80d\0\0\0\x0B\0\0\0\x0B\0\0\0\x0C\0\0\0\x0C\0\0\0") } ,)) , two : None , few : None , many : None , } ; static CY : < icu_plurals :: provider :: OrdinalV1Marker as icu_provider :: DataMarker > :: Yokeable = icu_plurals :: provider :: PluralRulesV1 { zero : Some (icu_plurals :: rules :: runtime :: ast :: Rule (unsafe { zerovec :: VarZeroVec :: from_bytes_unchecked (b"\x01\0\0\0\0\0\xC0\0\0\0\0\0\0\0\0\0\0\0\0\x07\0\0\0\x07\0\0\0\x08\0\0\0\x08\0\0\0\t\0\0\0\t\0\0\0") } ,)) , one : Some (icu_plurals :: rules :: runtime :: ast :: Rule (unsafe { zerovec :: VarZeroVec :: from_bytes_unchecked (b"\x01\0\0\0\0\0\xC0\0\0\0\0\x01\0\0\0\x01\0\0\0") } ,)) , two : Some (icu_plurals :: rules :: runtime :: ast :: Rule (unsafe { zerovec :: VarZeroVec :: from_bytes_unchecked (b"\x01\0\0\0\0\0\xC0\0\0\0\0\x02\0\0\0\x02\0\0\0") } ,)) , few : Some (icu_plurals :: rules :: runtime :: ast :: Rule (unsafe { zerovec :: VarZeroVec :: from_bytes_unchecked (b"\x01\0\0\0\0\0\xC0\0\0\0\0\x03\0\0\0\x03\0\0\0\x04\0\0\0\x04\0\0\0") } ,)) , many : Some (icu_plurals :: rules :: runtime :: ast :: Rule (unsafe { zerovec :: VarZeroVec :: from_bytes_unchecked (b"\x01\0\0\0\0\0\xC0\0\0\0\0\x05\0\0\0\x05\0\0\0\x06\0\0\0\x06\0\0\0") } ,)) , } ; * [& AF , & AF , & AF , & AS , & AZ , & BE , & AF , & AS , & AF , & CA , & AF , & CY , & AF , & AF , & AF , & EN , & AF , & AF , & AF , & AF , & AF , & FIL , & FIL , & FIL , & GD , & AF , & GU , & AF , & GU , & AF , & HU , & FIL , & AF , & AF , & IT , & AF , & KA , & KK , & AF , & AF , & AF , & AF , & FIL , & AF , & AF , & MK , & AF , & AF , & MR , & FIL , & AF , & NE , & AF , & AF , & OR , & AF , & AF , & AF , & AF , & FIL , & AF , & AF , & AF , & AF , & AF , & SQ , & AF , & SV , & AF , & AF , & AF , & AF , & TK , & AF , & UK , & AF , & AF , & FIL , & AF , & AF , & AF] . get_unchecked (i) }) . map (icu_provider :: AnyPayload :: from_static_ref) , _ => return Err (icu_provider :: DataErrorKind :: MissingDataKey . with_req (key , req)) , } . map (| payload | icu_provider :: AnyResponse { payload : Some (payload) , metadata : Default :: default () , }) . ok_or_else (|| icu_provider :: DataErrorKind :: MissingLocale . with_req (key , req)) } } } } # [doc (inline)] pub use __impl_any_provider as impl_any_provider ;
```

Comment:
issue: I would like us to prioritize fixing `rustfmt` on baked data; it's becoming really hard to figure out what's going on here.

I am very okay with having us use string interpolation instead

Reply:
Does it matter? You can review the config that generates this.

Reply:
Yes, because this macro world is still new and I want to know what I'm reviewing. It's becoming hard to grasp the full end-to-end implications of these PRs; and relatedly when working on https://github.com/unicode-org/icu4x/pull/3480/ I was having a ton of trouble understanding what was going on in the regen.

I can potentially look into this tomorrow if it's not high priority for you.